content/kubermatic/master/architecture/support_policy/provider_support_matrix/kubevirt/kubevirt.en.md

```diff
@@ -21,6 +21,32 @@ KKP supports KubeVirt Operator >= 0.19.0 and the Containerized Data Importer >=
 to run KubeVirt, however a Kubernetes cluster consists of 3 nodes with 2 CPUs, 4GB of RAM and 30GB of storage, to have a
 minimal installation.
 
+### KubeVirt Configuration Requirements
+Kubevirt requires the following configuration to be used with KKP.
+- In case your Kubevirt namespace has the ConfigMap 'kubevirt-config' then use this ConfigMap for adding the feature gates to it. Look at the path `{.data.feature-gates}`
+- Otherwise, add the feature gate to the resource of type Kubevirt. There should be a single resource of this type and its name is irrelevant
+
+The configuration KKP requires:
+```yaml
+apiVersion: kubevirt.io/v1
+kind: KubeVirt
+metadata:
+  name: kubevirt
+  namespace: kubevirt
+spec:
+  configuration:
+    developerConfiguration:
+      featureGates:
```

Comment:
Is the meaning of these feature gates obvious to everyone concerned or does it make sense to add a description to the documentation?

Reply:
I've added a link the the KubeVirt documentation.

Reply:
Top, thanks.